template/private-npm-package/README.md

```diff
@@ -120,3 +120,7 @@ For more information, see the **semantic-release** docs on [triggering a release
 [seek-jobs/renovate]: https://github.com/SEEK-Jobs/renovate
 [semantic-release]: https://github.com/semantic-release/semantic-release
 [triggering a release]: https://github.com/semantic-release/semantic-release/#triggering-a-release
+
+In order to facilitate the alignment with semantic release commit naming convention you can use `yarn commit` command. The script will take you through a range of questions, in the end of which you will have a correct commit name and description for semantic release to pick it up.
+
+However, if you are using _squash and merge_ commits you will need to ensure the convention is preserved upon merging the pull request on Github by ensuring title and description are still populated with necesary details and follow the convention.
```

Comment:
Should this go in its own section under `Release`? Something like:

```markdown
## Release

...

### Commit messages

Here!

### Releasing latest

...
```

Reply:
As you wish, I think it makes sense 👌